edxlearndot/management/commands/update_learndot_enrolments.py

```diff
@@ -78,7 +87,10 @@ def handle(self, *args, **options):
 
             log.info("Processing enrollments in course %s", cm.edx_course_key)
 
-            enrollments = CourseEnrollment.objects.filter(course=cm.edx_course_key)
+            enrollments = CourseEnrollment.objects.filter(course_id=cm.edx_course_key)
+            if options["users"]:
+                enrollments = enrollments.filter(user__username__in=options["users"])
+
```

Comment:
@ciuin  I'm afraid I didn't review the management command, which is my mistake.  But now that I have, it bugs me that there's so much code duplication between it and [`edxlearndot.signals. listen_for_passing_grade `](https://github.com/open-craft/edx-learndot/blob/ginkgo/edxlearndot/signals.py).  Could you factor out the duplication into a new method on [`edxlearndot.learndot`](https://github.com/open-craft/edx-learndot/blob/ginkgo/edxlearndot/learndot.py)?